src/utils/checkItemWishlist.js

```diff
@@ -0,0 +1,13 @@
+export const checkItemWishlist = (arr, product) => {
+  if (arr.length === 0) {
+    return false;
+  }
+
+  for (let item of arr) {
+    if (item._id === product._id) {
+      return true;
+    }
+  }
```

Comment:
you may use ES6 array methods

Reply:
Hey! thanks for pointing it out. Used arr.some() instead.